geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables/SpatioTemporalTable.scala

```diff
@@ -31,7 +31,7 @@ object SpatioTemporalTable extends GeoMesaTable with LazyLogging {
   val INDEX_CHECK = s"~$INDEX_FLAG~"
   val DATA_CHECK = s"~$DATA_FLAG~"
 
-  override def supports(sft: SimpleFeatureType): Boolean = true
+  override def supports(sft: SimpleFeatureType): Boolean = sft.getGeometryDescriptor != null
```

Comment:
wonder if we should also have sft.getDtgField.isDefined ?not sure if we need to check that or not...